src/reducers.js

```diff
@@ -0,0 +1,28 @@
+import {SET_LANGUAGES, SET_SCALE, SET_SCREENS, SET_URL, RESET_STATE} from './action-types';
+import {LANGUAGES, SCREENS} from './App';
+const initialState = {
+  scale: 50,
+  screens: [SCREENS[0], SCREENS[1]],
+  languages: [LANGUAGES[0], LANGUAGES[1]],
+  // src: "https://www.pexels.com"
+  src: 'http://www.bbc.com',
+};
+
+const rootReducer = (state = initialState, action) => {
```

Comment:
before trying `redux-actions` try to split `rootReducers` to smaller ones and then combine them with `combineReducers`